src/components/physics-section.svelte

```diff
@@ -372,14 +380,22 @@
             form={i + item._uid}
             block={item}
             bind:ref={refs[i]}
-            class={clsx('absolute w-fit', !engine && 'invisible')}
+            class={clsx(
+              'absolute w-fit',
+              !engine && 'invisible',
+              item.isDesktopOnly && 'invisible md:visible'
+            )}
           />
         </form>
       {:else if item.component === 'physics-sticker' && item.photo}
         {@const { width, height, src, alt } = getImageAttributes(item.photo)}
         <img
           bind:this={refs[i]}
-          class={clsx('absolute select-none drop-shadow-md', !engine && 'invisible')}
+          class={clsx(
+            'absolute select-none drop-shadow-md',
+            !engine && 'invisible',
+            item.isDesktopOnly && 'invisible md:visible'
```

Comment:
Wondering if `invisible` rather than `display: none` will make the `Engine` render a body even though we can't see it, the idea is to not overfill the container on small devices so please double check that.